File: Serene3/Serene3.Web/Modules/com/review/reviewForm.cs
using Serenity.ComponentModel;

namespace Serene3.com.Forms;

[FormScript("com.review")]
[BasedOnRow(typeof(reviewRow), CheckNames = true)]
public class reviewForm
{
    public string Name { get; set; }
    public string Email { get; set; }
    public string Subject { get; set; }
    public string Description { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/reviewForm.ts
import { initFormType, PrefixedContext, StringEditor } from "@serenity-is/corelib";

export interface reviewForm {
    Name: StringEditor;
    Email: StringEditor;
    Subject: StringEditor;
    Description: StringEditor;
}

export class reviewForm extends PrefixedContext {
    static readonly formKey = 'com.review';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!reviewForm.init) {
            reviewForm.init = true;

            var w0 = StringEditor;

            initFormType(reviewForm, [
                'Name', w0,
                'Email', w0,
                'Subject', w0,
                'Description', w0
            ]);
        }
    }
}